pkg/kubecost/allocationfilter.go

```diff
@@ -35,6 +37,7 @@ type FilterOp string
 const (
 	FilterEquals    FilterOp = "equals"
 	FilterNotEquals          = "notequals"
+	FilterContains           = "contains"
```

Comment:
Do you see `contains` being used for any other fields, or just "services"? (There is, of course, the idea of a string containing another string, for instance.)